Project.toml

```diff
@@ -1,20 +1,28 @@
 name = "TestSetExtensions"
 uuid = "98d24dd4-01ad-11ea-1b02-c9a08f80db04"
-authors = []
-version = "2.0.0"
+authors = ["Spencer Russell", "Phillip Alday"]
+version = "3.0.0"
 
 [deps]
 DeepDiffs = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-julia = "1"
+Aqua = "0.8"
 DeepDiffs = "1"
+Distributed = "<0.0.1, 1"
+Logging = "<0.0.1, 1"
+MetaTesting = "0.1"
+Suppressor = "0.2"
+Test = "<0.0.1, 1"
+julia = "1.6"
 
 [extras]
+Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+MetaTesting = "9e32d19f-1e4f-477a-8631-b16c78aa0f56"
```

Comment:
How much do we gain from MetaTesting? It doesn't look like the `EncasedTestSet` type you're using here is documented. AFAICT it is supposed to isolate the wrapped tests, but we still end up wrapping them in `try...catch` blocks.

I'm not strongly opposed, if you think it makes things easier. I just want to make sure there's a good reason to add an additional dependency.

Reply:
It made some of the testing easier -- without MetaTesting, the blocks were even more of an mess. It's "just" a test dep, so it doesn't hurt most users.

I can also set the compat entry for MetaTesting to exact equality, since we are reaching into internals a bit.

Reply:
sounds good. just wanted to make sure. re: making the compat entry exact, I'd say it's not necessary, but is good to keep in mind if things break in the future.

Reply:
whoops, didn't check the rest of the new commits before commenting. I'm good with it either way.